wagtail/admin/tests/test_rich_text.py

```diff
@@ -89,10 +88,59 @@ def setUp(self):
 
         self.login()
 
+    @override_settings()
+    def test_default_editor_in_rich_text_field(self):
+        # Simulate the absence of a setting
+        if hasattr(settings, 'WAGTAILADMIN_RICH_TEXT_EDITORS'):
+            del settings.WAGTAILADMIN_RICH_TEXT_EDITORS
+
+        response = self.client.get(reverse(
+            'wagtailadmin_pages:add', args=('tests', 'defaultrichtextfieldpage', self.root_page.id)
+        ))
+
+        # Check status code
+        self.assertEqual(response.status_code, 200)
+
+        # Check that draftail (default editor) initialisation is applied
+        self.assertContains(response, "window.draftail.initEditor('body',")
+
+        # check that media for draftail is being imported
+        self.assertContains(response, 'wagtailadmin/js/draftail.js')
+
+    @override_settings()
```

Comment:
What’s the point of overriding no setting?
Does this restores the `WAGTAILADMIN_RICH_TEXT_EDITORS` setting that we remove inside the method?
If so, adding a comment would make it less weird.